README.md

```diff
@@ -601,6 +601,15 @@ golang:
    # if unset this defaults to $GONOPROXY
    # SYFT_GOLANG_NOPROXY env var
    no-proxy: ""
+  
+java:
+   # when running across pom.xml files that could have more information, syft will
+   # explicitly search maven for license information by querying the online pom when this is true eg:
+   # https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-starter-test/3.1.5/spring-boot-starter-test-3.1.5.pom
+   # this option is helpful for when the parent pom has this information,
+   # but it is not accessible from within the final built artifact
+   search-maven-for-licenses: false
+   maven-central-url: "https://repo1.maven.org/maven2"
```

Comment:
maybe change this to `maven-url` and we can leave the default to point to central. There are multiple maven repos that exist, but there is only one `central`.